### ISSUE TITLE
Increase Sprite and Pseudodragon power

### ISSUE THREAD

**Issue by Marcloure** [mlenser/kryx-rpg-issues#977]
https://www.kryxrpg.com/feats/improved-familiar

When comparing the creatures offered by this feat, the imp and the quasit are greatly superior options. They have more mana, damage resistance, spell resistance, darkvision, shapeshifting abilitites, etc. Most of this stuff are inherent traits of the creature, so that is fine, but they also have 2 mana while the Sprite and Pseudodragon have only 1 mana. Since mana recovered on a short rest rounds down, the Imp and the Quasit have quite a lot more mana (and thus utility) than the other two options.

Scaling the Pseudodragon and the Sprite to Difficulty 1 should solve this issue.

**Comment by mlenser** [comment]
- **Creatures**
  - Gazer, Pseudodragon, and Sprite rebalanced to challenge ½ mainly by adding a health die and changing their spellcasting ability to 2. [#977](https://github.com/mlenser/kryx-rpg-issues/issues/977)
  - Quasit charisma increased to 2 and its poison damage is reduced. [#977](https://github.com/mlenser/kryx-rpg-issues/issues/977)

**Comment by mlenser** [comment]
Note: Quasit and Imp are full casters while the Pseudodragon, Sprite, and Gazer are not.

Imp and Quasit are inherently stronger due to the resistances. I don't think buffing everything to challenge 1 is the best option. This feat should probably be split:

- Gazer to the Mind theme.
- Imp and Quasit to the Unholy theme.
- Pseudodragon to the primal damage themes.
- Sprite to the Animal/Plants themes.

**Comment by Marcloure** [comment]
> Quasit and Imp are full casters while the Pseudodragon, Sprite, and Gazer are not.

Are Sprites not full casters? I mean, surely they don't have a strong attack, since they deal 1 damage with their weapon attack, and the imp deals 1+2d6

**Comment by mlenser** [comment]
> Are Sprites not full casters? I mean, surely they don't have a strong attack, since they deal 1 damage with their weapon attack while the imp deals 1+2d6. Not that attack damage matters for familiars, but it is strange to set the Sprite as half-caster when they have such a weak attack.

I saw what you're raising now and I don't have a solution for it. Sprites I see as a Ranger so it's not a full caster. Quasit and Imp should maybe be half as well since they have such strong attacks.

**Comment by Marcloure** [comment]
I think the gazer is a bit too powerful as a familiar. Its rays, which target 2 creatures per use, can cause restraint, fear, charm that gives disadvantage on attack rolls, or slow, and they can be used at-will. Sure, the effect is random, but any of them can be very powerful if they hit.

**Comment by mlenser** [comment]
> I think the gazer is a bit too powerful as a familiar.

Sounds like you're making an argument that Gazer is too powerful for challenge ½ or 1.

**Comment by Marcloure** [comment]
Not really, but there is a difference between familiars and normal creatures: familiars can't attack, but they can use saving throw options. Using the gazer rays is similar to allowing the imp to make attacks.

**Comment by mlenser** [comment]
A Gazer familiar shouldn't be able to use its eye rays, I agree.
